<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditBookDialog</class>
 <widget class="QDialog" name="EditBookDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Автор</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="authorLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Название</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="titleLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Жанр</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="genreLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Год</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="yearSpinBox">
     <property name="minimum">
      <number>1800</number>
     </property>
     <property name="maximum">
      <number>3000</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Издатель</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="publisherLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>ISBN</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="isbnLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Страницы</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="pageCountSpinBox"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
